cmd/kind/delete/cluster/deletecluster.go

```diff
@@ -45,6 +47,7 @@ func NewCommand() *cobra.Command {
 		},
 	}
 	cmd.Flags().StringVar(&flags.Name, "name", cluster.DefaultName, "the cluster name")
+	cmd.Flags().BoolVarP(&flags.Force, "force", "f", false, "skip the confirmation prompt")
```

Comment:
Do similar tools require this in non-interactive mode?

I also have some limited concern that this encourages the wrong way of thinking about kind clusters -- they're meant to be highly disposable and shouldn't be storing any critical state. Accidentally deleting one should be OK.

Reply:
kubeadm does it for `reset` nowadays.

i see the possibility of kind being used for local development where a accidental delete can remove one's progress with no undo.

Reply:
> kubeadm does it for reset nowadays.

fair, but `kubeadm reset` seems like a different class of command. these clusters may not be ephemeral. 
It doesn't look like `minikube delete` has any confirmation logic.

> i see the possibility of kind being used for local development where an accidental delete can remove one's progress with no undo.

Hmm, I understand the sentiment but... what progress? That still sounds like important state is being preserved inside the kind cluster, rather than just a copy being tested on it, which is a practice we should discourage.

When we have #62 (expect a form of it by ~monday) preserving data this way should be preferable to ensuring the cluster continues to exist.

This also means that even the simplest CI scripts (`kind create cluster && do_my_test || kind delete cluster`) now need a flag to work correctly and will otherwise hang indefinitely. 🤔

Reply:
we could do a quick poll in `#kind`

Reply:
WDYT about moving this commit to a second PR?

Reply:
sure thing.

Reply:
separate PR sent here: https://github.com/kubernetes-sigs/kind/pull/261